tutorial/README.md

```diff
@@ -6,6 +6,10 @@ These tutorials introduce to use the opensource Surrogate Modeling Toolbox where
 
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/SMTorg/smt/blob/master/tutorial/SMT_Tutorial.ipynb)
 
+## SMT Educational (How to start & begginers guidelines. )

```

Comment:
typo "beginners". Did not open the notebook but it seems to be focused on (bayesian) optimization EGO SBO, the title should reflects that.